src/towncrier/templates/default.md

```diff
@@ -15,7 +15,6 @@
 {% for category, val in definitions.items() if category in sections[section] %}
 ### {{ definitions[category]['name'] }}
 
-{% if definitions[category]['showcontent'] %}
```

Comment:
I think that we should add an `612.removal` fragment, informat that when `showcontent = False` the template will no longer receive the fragment content.

This is a note, just in case someone was missusing this feature with a custom template.

Reply:
I think there's some confusion here. The existing behaviour is already that fragment content is not sent to the template (https://github.com/twisted/towncrier/pull/612/files#diff-36db088409efe07783d49792a18cec6ae3778ace338bf1cd08daa8a3fce29e93L166). This change just means that orphan fragment contents **are** sent.

Reply:
True. My bad. I misinterpreted the changes in the code.